src/styles/delete.css

```diff
@@ -0,0 +1,94 @@
+.delete-tour {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 60px;
+  margin: 0 auto;
+  height: 100vh;
+  flex: 1;
+}
+
+.tour-container {
+  display: grid;
+  padding: 10px;
+  grid-template-columns: repeat(3, 1fr);
+  grid-auto-rows: 1fr;
+}
+
+.tour-element {
+  text-align: center;
+  position: relative;
+  color: white;
+  width: 250px;
+  height: 250px;
+}
+
+.tour-container img {
+  height: 100%;
+  width: 100%;
+  object-fit: cover;
+}
+
+.tour-detail {
+  width: 85%;
+  line-height: 25px;
+  letter-spacing: 0.07em;
+  color: #b7b3b3;
+  font-size: 14px;
+  margin-bottom: 4rem;
+}
+
+.midle {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  background-color: rgba(0, 0, 0, 0.3);
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.midle:hover {
+  background-color: rgba(0, 0, 0, 0.7);
+}
+
+.midle p {
+  margin: 1px 0;
+}
+
+.midle button {
+  padding: 10px;
+  width: fit-content;
+  border-radius: 50%;
+  background-color: inherit;
+  color: white;
+  font-weight: 900;
+  border: 1px solid white;
+}
+
+.disable {
+  display: none;
+}
+
+.delete-btn {
+  cursor: pointer;
+}
+
+@media (max-width: 950px) {
+  .tour-container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (max-width: 680px) {
+  .tour-container {
+    grid-template-columns: 1fr;
+  }
+
+  .delete-container {
+    height: 100%;
+  }
+}
```

Comment:
Please consider reviewing your styles as they don't seem to be working as expected. For more details I have created a short video below you can look at it.

https://user-images.githubusercontent.com/94223367/197714827-4e68b7b6-fb7b-4935-a972-82b32ed3ed46.mp4

Reply:
> hello @ABDULALI3468 cause we haven't yet linked the pages to the nav that's why as we are still accessing the through the routes that's why I left it that way and to be fixed after

@divinecharlotte I suggest you to add the routes even we are not implementing the Navbar for now. But for example if someone is manually going to delete tour page the style will broke. You can implement it by making it `position:fixed; top:0; right:0`. Hope that helps.